core/engine/src/vm/opcode/call/mod.rs

```diff
@@ -258,6 +258,92 @@ impl Operation for CallSpread {
     const COST: u8 = 3;
 }
 
+/// Parses the import attributes from the options object.
+fn parse_import_attributes(
```

Comment:
Suggestion: If this function implements the `https://tc39.es/ecma262/#sec-evaluate-import-call` (` EvaluateImportCall `), then we should call it evaluate_import_call